portal-ui/src/screens/Console/Buckets/ListBuckets/AddBucket/BucketNamingRules.tsx

```diff
@@ -56,24 +56,38 @@ const BucketNamingRules = ({ errorList }: { errorList: boolean[] }) => {
     uniqueName,
   ] = errorList;
 
+  const toggleNamingRules = () => {
+    setShowNamingRules(!showNamingRules);
+  };
+
   return (
     <Fragment>
       <Grid item xs={12}>
         {showNamingRules ? (
-          <span style={{ color: "#0288D1", textDecoration: "underline" }}>
+          <span
+            onClick={() => {
+              toggleNamingRules();
+            }}
+            style={{ color: "#0288D1", textDecoration: "underline" }}
+          >
             {" "}
             Hide Bucket Naming Rules{" "}
```

Comment:
@jinapurapu Not directly related to this PR, however I think we can get rid of the spaces here:

```suggestion
            Hide Bucket Naming Rules
```